app/controllers/application_controller.rb

```diff
@@ -33,4 +34,8 @@ def require_active_account!
     def multitenant?
       Settings.multitenant
     end
+
+    def set_site_i18n_keys
+      I18n.backend.store_translations(:en, sufia: { product_name: Site.instance.application_name })
```

Comment:
This is hideous; hopefully with https://github.com/projecthydra/sufia/pull/1783, we can now override the `application_name` helper.